libcxx/test/std/numerics/c.math/hermite.pass.cpp

```diff
@@ -203,6 +208,7 @@ std::vector<T> get_roots(unsigned n) {
 
 template <class Real>
 void test() {
+#if !(defined(__MVS__) && !defined(__BFP__))
```

Comment:
Similar to the other comment: Could we make use of `std::numeric_limits<Real>::has_???_NaN` here?

E.g. https://en.cppreference.com/w/cpp/types/numeric_limits/has_quiet_NaN